contracts/events/LoanOpeningsEvents.sol

```diff
@@ -13,6 +13,8 @@ pragma solidity 0.5.17;
  * This contract contains the events for loan openings operations.
  * */
 contract LoanOpeningsEvents {
+	event ContractReplaced(address indexed changedBy, address indexed newTargetAddr, bytes32 indexed module);
```

Comment:
there is DRY design principle
repeating the same function is a signal to refactor
i suggest creating a ModulesCommonEvents.sol, place the function there contract and inherit from it to every module

Reply:
Done